concepticondata/conceptlists/Hsiu-2015-315.tsv

```diff
@@ -0,0 +1,317 @@
+Hsiu-2015-315-142	142	house	1252	HOUSE
+Hsiu-2015-315-143	143	human, person	683	PERSON
+Hsiu-2015-315-144	144	hunt, to	1435	HUNT
+Hsiu-2015-315-145	145	husband	1200	HUSBAND
+Hsiu-2015-315-146	146	I	1209	I
+Hsiu-2015-315-147	147	insect	620	INSECT
+Hsiu-2015-315-148	148	intestines	1991	INTESTINES
+Hsiu-2015-315-149	149	iron	621	IRON
+Hsiu-2015-315-150	150	jaw	798	JAW
+Hsiu-2015-315-151	151	kill	1417	KILL
+Hsiu-2015-315-152	152	knee	1371	KNEE
+Hsiu-2015-315-153	153	knife	1352	KNIFE
+Hsiu-2015-315-154	154	know	1410	KNOW (SOMETHING)
+Hsiu-2015-315-155	155	laugh	1355	LAUGH
+Hsiu-2015-315-156	156	leaf	628	LEAF
+Hsiu-2015-315-157	157	left	244	LEFT
+Hsiu-2015-315-158	158	leg	1297	LEG
+Hsiu-2015-315-159	159	lie down	215	LIE DOWN
+Hsiu-2015-315-160	160	light	737	LIGHT (IGNITE)
+Hsiu-2015-315-161	161	lightning	1654	LIGHTNING
+Hsiu-2015-315-162	162	light (not heavy)	1052	LIGHT (WEIGHT)
+Hsiu-2015-315-163	163	lip	478	LIP
+Hsiu-2015-315-164	164	live, to	1422	BE ALIVE
+Hsiu-2015-315-165	165	liver	1224	LIVER
+Hsiu-2015-315-166	166	long	1203	LONG
+Hsiu-2015-315-167	167	look, to	1819	LOOK
+Hsiu-2015-315-168	168	louse, body	1392	LOUSE
+Hsiu-2015-315-169	169	louse, head	310	HEAD LOUSE
+Hsiu-2015-315-170	170	male	1553	MALE
+Hsiu-2015-315-171	171	market	633	MARKET
+Hsiu-2015-315-172	172	meat	634	MEAT
+Hsiu-2015-315-173	173	midday	12	MIDDAY
+Hsiu-2015-315-174	174	millet	931	MILLET
+Hsiu-2015-315-175	175	money	1559	MONEY
+Hsiu-2015-315-176	176	monkey	1350	MONKEY
+Hsiu-2015-315-177	177	month	1370	MONTH
+Hsiu-2015-315-178	178	moon	1313	MOON
+Hsiu-2015-315-179	179	morning	1339	MORNING
+Hsiu-2015-315-180	180	mortar	224	MORTAR CRUSHER
+Hsiu-2015-315-181	181	mosquito	1509	MOSQUITO
+Hsiu-2015-315-182	182	mountain	639	MOUNTAIN
+Hsiu-2015-315-183	183	mouth	674	MOUTH
+Hsiu-2015-315-184	184	mud	640	MUD
+Hsiu-2015-315-185	185	mushroom	641	MUSHROOM
+Hsiu-2015-315-186	186	name	1405	NAME
+Hsiu-2015-315-187	187	narrow	1267	NARROW
+Hsiu-2015-315-188	188	navel	1838	NAVEL
+Hsiu-2015-315-189	189	near	1942	NEAR
+Hsiu-2015-315-190	190	neck	1333	NECK
+Hsiu-2015-315-191	191	needle	1382	NEEDLE (FOR SEWING)
+Hsiu-2015-315-192	192	new	1231	NEW
+Hsiu-2015-315-193	193	night	1233	NIGHT
+Hsiu-2015-315-194	194	nose	1221	NOSE
+Hsiu-2015-315-195	195	oil	1484	OIL (ORGANIC SUBSTANCE)
+Hsiu-2015-315-196	196	old (cu)	2113	OLD (USED)
+Hsiu-2015-315-197	197	old (gia)	2112	OLD (AGED)
+Hsiu-2015-315-198	198	open, to	1156	OPEN
+Hsiu-2015-315-199	199	oven	1143	OVEN
+Hsiu-2015-315-200	200	pig	1337	PIG
+Hsiu-2015-315-201	201	plant, to	1486	PLANT (SOMETHING)
+Hsiu-2015-315-202	202	plate	480	PLATE
+Hsiu-2015-315-203	203	plow	2154	PLOUGH (INSTRUMENT)
+Hsiu-2015-315-204	204	poor	1674	POOR
+Hsiu-2015-315-205	205	potato	593	POTATO
+Hsiu-2015-315-206	206	pound	354	POUND
+Hsiu-2015-315-207	207	rain	658	RAIN (PRECIPITATION)
+Hsiu-2015-315-208	208	rainbow	1733	RAINBOW
+Hsiu-2015-315-209	209	rat	1490	RAT
+Hsiu-2015-315-210	210	red	156	RED
+Hsiu-2015-315-211	211	rice, cooked	2750	GLUTINOUS RICE
```

Comment:
two times the same concept? Did you check the meanings by comparing forms for the varieties against other HM languages?

Reply:
please make sure to address this point.